README.md

```diff
@@ -1,6 +1,8 @@
 # parse_vNav_Motion.py
 
-This is a small python script to parse DICOM files from a vNav series and convert them into motion scores. The script takes four arguments.  The usage and help information below can be generated any time by calling it with the `-h` or `--help` flag.
+This is a small python package that exposes both functions for parsing/interpreting motion in vNavs DICOM files, and an executable that can handle the whole process. To install the package, use `pip install git+https://github.com/MRIMotionCorrection/parse_vNav_Motion#egg=vnav`
```

Comment:
How do folks feel about registering the package with PyPI, that way users can simply run `pip install vnav`? And other users could include this package as a dependency.

Looks like the hard work of packaging the repo has been done, and all that's left to do is: `python setup.py register`

[More info](https://python-packaging.readthedocs.io/en/latest/minimal.html#publishing-on-pypi)

Reply:
That's definitely what we're working towards. However, there's still a few more bugs that I think we should fix before we mark it as "released". I'm working on those bugs right now, but I figured I'd like to leave the documentation in a state that matches what's in the repos right now. That way, if I end up not being able to commit fixes for weeks/months, the repos is at least self-consistent.